gems/spree/OSVDB-91217.yml

```diff
@@ -0,0 +1,17 @@
+---
+gem: spree
+cve: 2013-1656
+osvdb: 91217
+url: https://spreecommerce.com/blog/multiple-security-vulnerabilities-fixed
+title: |
+  Spree payment_methods_controller.rb payment_method Parameter Arbitrary Ruby
+  Object Instantiation Command Execution
+date: 2013-02-21
+description: |
+  Spree contains a flaw that is triggered when handling input passed via the
+  'payment_method' parameter to payment_methods_controller.rb. This may allow
+  a remote authenticated attacker to instantiate arbitrary Ruby objects and
+  potentially execute arbitrary commands.
+cvss_v2: 4.3
+patched_versions:
+  - ">= 2.0.0"
```

Comment:
The CVE says the affected versions are 1.0.x-1.3.2 so wouldn't the patched version be 1.3.3? Or did you research more and find it was 2.0.0?